mabwiser/mab.py

```diff
@@ -5,7 +5,7 @@
 """
 :Author: FMR LLC
 :Email: mabwiser@fmr.com
-:Version: 1.6.3 of November 14, 2019
+:Version: 1.7.0 of November 27, 2019
```

Comment:
Version attribute below needs to be updated to 1.7.0

Reply:
I always forget that one! Thanks for catching it